src/Resie.jl

```diff
@@ -62,69 +62,91 @@ function run_simulation(project_config::Dict{AbstractString,Any})
     )
     EnergySystems.set_timestep(parameters["time_step_seconds"])
 
```

Comment:
The function run_simulation was already a bit long before the changes in this PR, but I think it might be time to move the functionality for gathering outputs into smaller functions to improve readability.

Reply:
I thought the same...